server/chat-plugins/mafia.ts

```diff
@@ -2547,26 +2575,35 @@ export const commands: Chat.ChatCommands = {
 			}
 			switch (cmd) {
 			case 'idle':
-				player.idle = true;
+				player.action = false;
 				user.sendTo(room, `You have idled.`);
 				break;
 			case 'action':
-				player.idle = false;
+				player.action = true;
 				if (target) {
-					this.errorReply(`'/mafia action' should be sent alone, not followed by your action or target. Please PM your exact action to the host.`);
+					player.action = target;
+					try {
+						this.checkBanwords(room, target);
+					} catch (e: any) {
```

Comment:
```suggestion
					} catch {
```
Since the `e` is unused, it can be omitted.